cmd/livepeer/livepeer.go

```diff
@@ -268,10 +270,8 @@ func main() {
 		n.NodeType = core.TranscoderNode
 	} else if *broadcaster {
 		n.NodeType = core.BroadcasterNode
-	} else if *redeemer {
-		n.NodeType = core.RedeemerNode
 	} else {
-		glog.Fatalf("Node type not set; must be one of -broadcaster, -transcoder or -orchestrator")
+		glog.Info("Node type not set; must be one of -broadcaster, -transcoder, -orchestrator or -redeemer")
```

Comment:
Let's remove the "must be one of" part because that is no longer accurate.

An example updated message could be:

"Node type not set with -broadcaster, -transcoder, -orchestrator or -redeemer"